cfgov/unprocessed/js/modules/SkipNav.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Applies a focus state to the main content when a user clicks the skip navigation link
+ * Improves the usability of the skip navigation link across browsers & devices
+ */
+
+function SkipNav() {
+	const skip = document.getElementById( 'skip-nav' );
+	const mainContent = document.getElementById( 'main' );
+
+	skip.addEventListener( 'click', function( e ) {
+    e.preventDefault();
+    mainContent.setAttribute( 'tabindex', '-1' );
+    mainContent.focus();
+	}, false );
+}
+
+module.exports = SkipNav;
```

Comment:
TRIVIAL: Add trailing newline to file (`gulp lint --fix` likely will capture this, I can't remember).

Reply:
Addressed in latest commit